data/telegraf_plugins.yml

```diff
@@ -156,6 +156,14 @@ input:
     introduced: 1.12.0
     tags: [linux, macos, windows, systems]
 
+  - name: Azure Storage Queue
+    id: azure_storage_queue
+    description: |
+      The Azure Storage Queue plugin gathers sizes of Azure Storage Queues.
+    link: https://github.com/influxdata/telegraf/blob/master/plugins/inputs/azure_storage_queue/README.md
```

Comment:
Can we link to the `release-X.Y` branch for all readme's?   At one point we had this, I guess it was lost a few versions back.  This avoids users getting the development documentation on master and trying to use new features before they are released.

Reply:
Good call. I did this initially because not all plugin links followed the same pattern. I've updated the template to auto-generate the Github URLs _unless_ a `link` attribute is present. This attribute overrides the link template. There are only a few of these.